### PR TITLE
Create Municipal Boundaries Final update for 2022

### PR DESCRIPTION
**Pull request by mheagin**
Yearly update for Municipal Boundaries

### CODE REVIEW

**Comment by gregbunce** [comment]
🎉

File: _posts/Municipal Boundaries Final update for 2022 [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+date: 2023-01-26 15:58:20
+categories:
+  - SGID Blog
+tags:
+  - municipal boundaries
+  - Munis
+  - annexations
+  - city boundaries
+---
+End of another long year and, ready or not, 2023 is here. Another year of growth for the state which has slowed down some.Top contributors are Utah and Cache County once again with 23 annexations in Utah Co., and another 16 in Cache Co.. Five other counties follow with 5 to 7 annexations this year. There is tie for the biggest growing city, both had six annexations. Utah County’s, Lehi and Sevier County’s, Richfield. The State’s  total for the year is 90 annexations this year; 84 annexations, 3 disconnections, 2 adjustments and one incorporation. This is similar to the last couple of years.
```
Comment (stdavis) [inline _posts/Municipal Boundaries Final update for 2022:15]:
```suggestion
It's the end of another long year and, ready or not, 2023 is here. Another year of growth for the state which has slowed down some. Top contributors are Utah and Cache County once again with 23 annexations in Utah Co., and another 16 in Cache Co.. Five other counties follow with 5 to 7 annexations this year. There is a tie for the biggest growing city, both had six annexations: Utah County’s Lehi and Sevier County’s Richfield. The State’s total for the year is 90 annexations; 84 annexations, 3 disconnections, 2 adjustments, and one incorporation. This is similar to the last couple of years. 
```

Just some grammatical suggestions...

**Comment by steveoh** [comment]
@mheagin did you use the [templater](https://agrc.github.io/templater/)? The file name is not valid.

**Comment by steveoh** [comment]
@mheagin this looks to have all of the same problems that I thought you and I went through and updated your notes to correct. What happened?

**Review by steveoh — APPROVED** [review]

**Pull request merged**